pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/NamespaceStats.java

```diff
@@ -72,6 +74,7 @@ public Metrics add(String namespace) {
         dMetrics.put("brk_no_of_consumers", consumerCount);
         dMetrics.put("brk_msg_backlog", msgBacklog);
         dMetrics.put("brk_replication_backlog", msgReplBacklog);
+        dMetrics.put("brk_replication_delay_second", msgReplDelayInSeconds);
```

Comment:
Should it be made explicit this is the max value? That would help people building aggregations in dashboard as well.